pegasus/sites.v3/code.org/views/inspirational_videos.haml

```diff
@@ -1,20 +1,22 @@
-- videos << { id: 'dontteach', code: 'nKIu9yen5nc', text: "What Most Schools Don't Teach", duration: '5 min', download: '//videos.code.org/social/what-most-schools-dont-teach.mp4', image: '/images/inspirational_videos_thumbnail_2.jpg'}
-- videos << { id: 'obama',     code: '6XvmhE1J9PY', text: 'President Obama on computer science', duration: '1 min', download: '//videos.code.org/2015/social/president_obama_asks_america.mp4', image: '/images/inspirational_videos_thumbnail_3.jpg'}
-- videos << { id: 'hochere',   code: 'FC5FbmsH4fw', text: 'Hour of Code is Here - Anybody Can Learn', duration: '2 min', download: '//s3.amazonaws.com/cdo-videos/HoC-video-15mb.mp4', image: '/images/inspirational_videos_thumbnail_4.jpg'}
-- videos << { id: 'learn', code: 'qYZF6oIZtfc', text: 'Anybody Can Learn', duration: '1 min', download: '//videos.code.org/social/codeorg-60sec.mp4', image: '/images/inspirational_videos_thumbnail_5.jpg'}
+- videos << { id: 'cs_changing_everything', code: 'QvyTEx1wyOY', text: 'Computer Science is Changing Everything', duration: '5 min', download: '//videos.code.org/social/cs-is-everything.mp4'}
+- videos << { id: 'girlpower', code: 'mFPg96gdPkc', text: 'Inspire a girl', duration: '2 min', download: '//videos.code.org/social/ChangeTheWorld_v5.mp4'}
+- videos << { id: 'csintro',   code: '_IiPLtP-jm8', text: 'Computer Science intro', duration: '1 min', download: '//videos.code.org/2015/social/hour_of_code_cs_matters.mp4'}
+- videos << { id: 'dontteach', code: 'nKIu9yen5nc', text: "What Most Schools Don't Teach", duration: '5 min', download: '//videos.code.org/social/what-most-schools-dont-teach.mp4'}
+- videos << { id: 'obama',     code: '6XvmhE1J9PY', text: 'President Obama on computer science', duration: '1 min', download: '//videos.code.org/2015/social/president_obama_asks_america.mp4'}
+- videos << { id: 'hochere',   code: 'FC5FbmsH4fw', text: 'Hour of Code is Here - Anybody Can Learn', duration: '2 min', download: '//s3.amazonaws.com/cdo-videos/HoC-video-15mb.mp4'}
+- videos << { id: 'learn', code: 'qYZF6oIZtfc', text: 'Anybody Can Learn', duration: '1 min', download: '//videos.code.org/social/codeorg-60sec.mp4'}
 
 - videos.each do |video|
   .col-50{style: "float:left; padding:10px"}
     - url = "https://www.youtube.com/watch?v=#{video[:code]}"
     - facebook = {u: url}
     - twitter = {url: url, related: 'codeorg', text: "#{video[:text]} @codeorg"}
-    =view :display_video_thumbnail, id: video[:id], video_code: video[:code], caption: "#{video[:text]} (#{video[:duration]})", play_button: 'center', facebook: facebook, twitter: twitter, download_path: video[:download], image: video[:image]
+    =view :display_video_thumbnail, id: video[:id], video_code: video[:code], caption: "#{video[:text]} (#{video[:duration]})", play_button: 'center', facebook: facebook, twitter: twitter, download_path: video[:download]
 
 %div{style: 'clear: both'}
-%br/
--if request.locale == 'en-US'
```

Comment:
I believe we put this because the string wasn't translated yet (for the new tutorial landing pages). We don't need to gate this anymore.